### DISCUSSION TITLE
[FTL] Could not create runner: could not find massdns binary

### DISCUSSION THREAD

**Discussion by VinzelOps** [projectdiscovery/shuffledns#221] [Q&A]
![image](https://user-images.githubusercontent.com/88572751/236646313-fd292ea4-5d01-4629-8ee0-c3ce2da51181.png)

[FTL] Could not create runner: could not find massdns binary

Please for help

**Comment by ehsandeep** [comment]
@VinzelOps see https://github.com/projectdiscovery/shuffledns#prerequisite
